src/components/Picker/Picker.js

```diff
@@ -3,13 +3,13 @@ import React, {PureComponent} from 'react';
 import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import RNPickerSelect from 'react-native-picker-select';
-import Icon from './Icon';
-import * as Expensicons from './Icon/Expensicons';
-import FormHelpMessage from './FormHelpMessage';
-import Text from './Text';
-import styles from '../styles/styles';
-import themeColors from '../styles/themes/default';
-import {ScrollContext} from './ScrollViewWithContext';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import FormHelpMessage from '../FormHelpMessage';
+import Text from '../Text';
+import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
+import {ScrollContext} from '../ScrollViewWithContext';
 
 const propTypes = {
```

Comment:
Since no props are specifically referenced in the platform-specific wrappers, it might be unnecessary to perform validation there before passing them to the BasePicker. However, if this is an established pattern in the codebase, it would be a good idea to follow that with our changes as well.

Reply:
I think I don't feel too passionate about this one. Or at least I can't think of a situation where it would help because the props are just getting passed straight to the BasePicker. If there was some prop validation warning we would see it either way.